main_app/templates/main_app/home.html

```diff
@@ -106,10 +113,17 @@
         border-left: 0.3em solid transparent;
       }
 
+  <div class="objectives" style="text-align: center;padding:60px;">
```

Comment:
- [ ] Kindly close the `}` and add this code below `gif image`. The HTML code can't be put into the `styles` tag. 
- [ ] Don't remove the font style. Just replace it with some other good font style that is readable.